roles/dev/tasks/main.yml

```diff
@@ -203,3 +203,6 @@
 
 - name: Enable SSP webapp
   file: src=/usr/local/share/simplesamlphp/ dest=/opt/www/{{ ssp_vhost_name }} state=link
+
+
+- include: db.yml
```

Comment:
Nice way to guarantee the creation of the test databases only occur in dev mode!